config/locales/views/credits/fr.yml

```diff
@@ -48,6 +48,8 @@ fr:
       tag: 'Tag: %{name}'
       status:
         user:
-          other: You have %{count} personal credits to spend
+          one: "You have %{count} personal credit to spend"
```

Comment:
Question: shouldn't this file be in French?

Reply:
Indeed - that's an opportunity for a french speaker! There are two layers to the translation project, one is infrastructure - providing points to customize the language, the other is content - providing localized strings in the correct language. This correcting the issue with the infrastructural concern only (that it's possible to have a single spendable credit, and you get an error in any language).